components/build-service/README.md

```diff
@@ -83,3 +83,25 @@ same `SharedSecret`. This is done by default. However, the `Secret` referred to
 `SharedSecret` may not exist at bootstrap time. This is ok. The underlying `Secret` can be created
 at a later time, and/or updated as needed. The changes should be reflected automatically within the
 Tekton Chains Controller without requiring a Pod restart.
+
+## Build Service secrets
+
+List of secrets:
+
+| Name | Source | Description |
+| -- | -- | -- |
+| pipelines-as-code-secret | appsre vault | Secret containg 'github-application-id', 'github-private-key' and 'webhook.secret' of GitHub Application used by Bild service for creating pull-requests |
+
+Rotation rule: Secrets must be rotated within 7 days after someone with access leaves the organization. Secret older than one year should be rotated.
+
+### Instruction for rotation of pipelines-as-code-secret
+
+Prerequisite:
+- User must have admin role on organization owning the GitHub Application.
+- GitHub Application is shared between Build team and Pipeline Service team - both teams must be aware of rotation
```

Comment:
GitHub Application  -> The GitHub Application 
both teams must be aware of rotation -> both teams must be aware of the rotation